catalogs/sources/gtfs/schedule/us-california-wasco-1782.json

```diff
@@ -0,0 +1,16 @@
+{
+    "mdb_source_id": 1782,
+    "data_type": "gtfs",
+    "provider": "City of Wasco",
+    "features": ["flex","flex-v2"],
+    "status": "development",
+    "location": {
+        "country_code": "US",
+        "subdivision_name": "California",
+        "municipality": "Wasco"
+        }
+    },
+    "urls": {
+        "direct_download": "https://gtfs.calitp.org/WascoDialaRideFlex.zip"
```

Comment:
I added it manually...

Reply:
Ok. The easiest way to create the `latest` URL would be to delete this file and to use the `tools.operations.add_gtfs_schedule_source` function to add the source. Otherwise, you can add the `latest` URL by taking example on another source. Without the `latest` URL, the source won't be stored every day.